string[] pending_nodes # Node names that didn't notify they are init or not yet.
string[] ready_nodes   # Node names that responded they initialized correctly.
string[] failed_nodes  # Node names that responded they couldn't init.